### PR TITLE
Packer implimentation

### PR DESCRIPTION
**Pull request by niallrobinson**

### CODE REVIEW

File: imageservice/networking.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ def postImage(img_data, data, field_width, field_height):
             metadata
     """
     tempfilep = os.path.join(tempfile.gettempdir(), "temp.png")
+
+    field_height, field_width = img_data.shape[:2]
     with open(tempfilep, "wb") as img:
         imageproc.writePng(img_data, img,
                   height=field_height, width=field_width*2,
```
Comment (RPrudden) [inline imageservice/networking.py]: I think this needs updating, writePng doesn't take height and width params any more.

**Comment by niallrobinson** [comment]
@RPrudden have a look at these fixes

File: imageservice/serveupimage.py [hunk 2]
```diff
@@ -59,7 +54,7 @@ def procTimeSliceToImage(
     # do any further processing (saturation etc) and convert to 8 bit uints
     proced_data = dataproc.procDataCube(rg_data)
 
-    data_tiled = imageproc.tileArray(proced_data.data, field_width, field_height)
+    data_tiled = imageproc.tileArray(proced_data.data)
     shadows_tiled = shadowproc.procShadows(data_tiled)
```
Comment (RPrudden) [inline imageservice/serveupimage.py]: procShadows now requires 2 arguments, second being the data shape

**Comment by niallrobinson** [comment]
btw - cudos to @Laurence-Cullen for this

**Pull request merged**